tests/unit/LHNOrderTest.js

```diff
@@ -0,0 +1,428 @@
+import {LocaleContextProvider} from '../../src/components/withLocalize';
+
+const fakeInsets = {
+    top: 0,
+    left: 0,
+    right: 0,
+    bottom: 0,
+};
+
+const fakePersonalDetails = {
+    'email1@test.com': {
+        login: 'email1@test.com',
+        displayName: 'Email One',
+        avatar: 'none',
+        firstName: 'ReportID',
+    },
+    'email2@test.com': {
+        login: 'email2@test.com',
+        displayName: 'Email Two',
+        avatar: 'none',
+        firstName: 'One',
+    },
+    'email3@test.com': {
+        login: 'email3@test.com',
+        displayName: 'Email Three',
+        avatar: 'none',
+        firstName: 'ReportID',
+    },
+    'email4@test.com': {
+        login: 'email4@test.com',
+        displayName: 'Email Four',
+        avatar: 'none',
+        firstName: 'Two',
+    },
+    'email5@test.com': {
+        login: 'email5@test.com',
+        displayName: 'Email Five',
+        avatar: 'none',
+        firstName: 'ReportID',
+    },
+    'email6@test.com': {
+        login: 'email6@test.com',
+        displayName: 'Email Six',
+        avatar: 'none',
+        firstName: 'Three',
+    },
+};
+
+const fakeReport1 = {
+    reportID: 1,
+    reportName: 'Report One',
+    unreadActionCount: 0,
+
+    // This report's last comment will be in the past
+    lastMessageTimestamp: Date.now() - 3000,
+    participants: ['email1@test.com', 'email2@test.com'],
+};
+const fakeReport2 = {
+    reportID: 2,
+    reportName: 'Report Two',
+    unreadActionCount: 0,
+    lastMessageTimestamp: Date.now() - 2000,
+    participants: ['email3@test.com', 'email4@test.com'],
+};
+const fakeReport3 = {
+    reportID: 3,
+    reportName: 'Report Three',
+    unreadActionCount: 0,
+    lastMessageTimestamp: Date.now() - 1000,
+    participants: ['email5@test.com', 'email6@test.com'],
+};
+
+const fakeReport1Actions = {
+    actionName: 'ADDCOMMENT',
+    person: [],
+    sequenceNumber: 0,
+
+    // This comment will be in the past
+    timestamp: Date.now() - 2000,
+    message: [
+        {type: 'comment', reportID: 1, text: 'Comment One'},
+    ],
+};
+const fakeReport2Actions = {
+    actionName: 'ADDCOMMENT',
+    person: [],
+    sequenceNumber: 0,
+    timestamp: Date.now() - 1000,
+    message: [
+        {type: 'comment', reportID: 2, text: 'Comment Two'},
+    ],
+};
+const fakeReport3Actions = {
+    actionName: 'ADDCOMMENT',
+    person: [],
+    sequenceNumber: 0,
+    timestamp: Date.now(),
+    message: [
+        {type: 'comment', reportID: 2, text: 'Comment Three'},
+    ],
+};
+
+const ONYXKEYS = {
+    PERSONAL_DETAILS: 'personalDetails',
+    CURRENTLY_VIEWED_REPORTID: 'currentlyViewedReportID',
+    NVP_PRIORITY_MODE: 'nvp_priorityMode',
+    COLLECTION: {
+        REPORT: 'report_',
+        REPORT_ACTIONS: 'reportActions_',
+    },
+};
+
+Onyx.init({
+    keys: ONYXKEYS,
+    registerStorageEventListener: () => {},
+});
+
+jest.disableAutomock();
```

Comment:
Can we add a comment explaining why we need this?

Reply:
I'm going to remove it actually. I only added this here to see if it fixed some of the test errors, and it had no effect.